### PR TITLE
Add a state management how to

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Add a state management how-to.

## Help wanted

* Does the example code make sense?
* Do we need to add anything?

## Review

* [Page to review](https://quixdocsdev.blob.core.windows.net/pr208/platform/how-to/state-management.html)

### CODE REVIEW

File: docs/platform/how-to/state-management.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# State management
+
+When deploying a service it is possible for that service to maintain state between restarts. This can be done by enabling state management for the deployment. Enabling state management creates a storage area that can be used for storage of data and files, and this state is preserved between service restarts.
```
Comment (SteveRosam) [inline docs/platform/how-to/state-management.md]: When deploying a service, enabling state management allows the service to maintain its state between restarts by creating a storage area for data and files, ensuring persistence across service restarts.

does this read better?

File: docs/platform/how-to/state-management.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+## Enabling state management
+
+When you deploy a service, it is possible to enable state management:
```
Comment (SteveRosam) [inline docs/platform/how-to/state-management.md]: Enable state management during service deployment if your code relies on state or expects files to persist across restarts.

??

File: docs/platform/how-to/state-management.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+When deploying a service it is possible for that service to maintain state between restarts. This can be done by enabling state management for the deployment. Enabling state management creates a storage area that can be used for storage of data and files, and this state is preserved between service restarts.
+
+## Enabling state management
+
+When you deploy a service, it is possible to enable state management: 
+
+![state management](../images/how-to/state/state-management.png){width=80%}
+
+This creates a `state` folder in the service.
```
Comment (SteveRosam) [inline docs/platform/how-to/state-management.md]: Enabling state creates a `state` folder that can be accessed by your code.

**Review by SteveRosam — DISMISSED** [review]

File: docs/platform/how-to/state-management.md (hunk 1 above)
Comment (tbedford) [reply docs/platform/how-to/state-management.md]: Yes!

**Review by SteveRosam — APPROVED** [review]

**Pull request merged**